VAMobile/src/store/slices/requestAppointmentSlice.ts

```diff
@@ -39,26 +39,31 @@ export const initialRequestAppointmentState: RequestAppointmentState = {
  * */
 export const getUserVAEligibility =
   (screenID?: ScreenIDTypes): AppThunk =>
-  async (dispatch, getState) => {
+  async (dispatch) => {
     dispatch(dispatchClearErrors(screenID))
     dispatch(dispatchSetTryAgainFunction(() => dispatch(getUserVAEligibility(screenID))))
-
-    try {
-      const facilities = filter(getState().patient.facilities || [], (item) => {
-        return item.isCerner === false
-      }).map((item) => item.facilityId)
-
-      dispatch(dispatchStartGeVAEligibility())
-      const eligibilityData = await api.get<api.UserVAEligibilityData>('/v0/appointments/va/eligibility', {
-        'facilityIds[]': facilities,
-      })
-
-      dispatch(dispatchFinishGetVAEligibility({ eligibilityData: eligibilityData?.data.attributes.services, ccSupported: eligibilityData?.data.attributes.ccSupported }))
-    } catch (error) {
-      if (isErrorObject(error)) {
-        logNonFatalErrorToFirebase(error, 'getUserEligibility: Request Appointment Service Error')
-        dispatch(dispatchFinishGetVAEligibility({ error }))
-        dispatch(dispatchSetError({ errorType: getCommonErrorFromAPIError(error), screenID }))
+    const { data: facilitiesInfo } = useFacilitiesInfo()
+
+    if (facilitiesInfo) {
+      try {
+        const facilities = facilitiesInfo
```

Comment:
Same suggested change as above. Although since this is used in three different places, I'm thinking it should remain a shared function

Reply:
this is actually the opposite of the other 2. This one gets the non cerner facilities the others get the cerner facilities